pulumi/python/infrastructure/aws/eks/__main__.py

```diff
@@ -58,20 +58,16 @@ def retrieve_vpc_and_subnets(vpc) -> VPCDefinition:
 stack_ref = pulumi.StackReference(stack_ref_id)
 vpc_definition: pulumi.Output[VPCDefinition] = stack_ref.get_output('vpc').apply(retrieve_vpc_and_subnets)
 
-node_group_opts = eks.ClusterNodeGroupOptionsArgs(
```

Comment:
Is this part of the fix that to the VPC standup failing intermittently? Let's drop a comment here linking to the issue and explaining what we did.

Reply:
done.